pkgs/by-name/na/namida/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "namida";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "meew0";
+    repo = "namida";
+    rev = "782ab6baf679f830c4242bf071d8e85743fe77f7";
+    hash = "sha256-pflucKCzfM3Fc2+M8GNq3fsTCln+9bHBGb7/mv/GKVs=";
+  };
+  cargoHash = "sha256-FT5pxu40A8CSPL0pd1EVK+Ti80F3pdNvh5ojKNlz9Rc=";
+
+  patches = [
+    # Needed for rustc/cargo < 1.80
+    # std::mem::size_of
+    ./client_get_mem_size_of.patch
+
+    # Removes checking git for revision and generating build-time timestamp
+    # Provided by environment variables in this nixpkg instead
+    ./make_deterministic.patch
+  ];
+
+  # Set the git revision and build datetime declaritively
+  env = {
+    GIT_HASH = src.rev;
+    NAMIDA_COMPILE_DT = "2024-10-13 12:00:00"; # Date this package was committed
```

Comment:
Timestamp fixed.  

>Niche

It is a bit niche, but I think it may be a bit more fair to look at the interest in the Tsunami protocol [in general](https://github.com/search?q=tsunami+udp&type=repositories&s=stars&o=desc).  It is used a bit more in internal data center networks and other situations where TCP can be congested or otherwise constrained (satellite ISPs for example).

Neither use case is likely to generate much activity in public repositories.